src/test/lib/scheduler/scheduler_test.cpp

```diff
@@ -126,113 +132,137 @@ TEST_F(SchedulerTest, BasicTest) {
   Hyrise::get().topology.use_fake_numa_topology(8, 4);
   Hyrise::get().set_scheduler(std::make_shared<NodeQueueScheduler>());
 
-  std::atomic_uint32_t counter{0};
+  auto counter = std::atomic_uint32_t{0};
 
   increment_counter_in_subtasks(counter);
 
   Hyrise::get().scheduler()->finish();
 
-  ASSERT_EQ(counter, 30u);
+  ASSERT_EQ(counter, 30);
 
   Hyrise::get().set_scheduler(std::make_shared<ImmediateExecutionScheduler>());
```

Comment:
Does this actually test changing the scheduler (would expect sth like that in `hyrise_test.cpp`) or should this somehow ensure we use the ImmediateExecutionScheduler afterward? If it's the latter, we can get rid of it because the test dtor calls `Hyrise::reset()`.

Reply:
I just removed it and added a simple test to the hyrise singleton test.